_docs-sources/foundations/terraform/folder_structure.md

```diff
@@ -1,5 +1,14 @@
 # Recommended Folder Structure
 
+TODO: Harmonize this content with:
+- https://github.com/orgs/gruntwork-io/discussions/769
+- https://github.com/orgs/gruntwork-io/discussions/765
+- https://github.com/orgs/gruntwork-io/discussions/756
+
+Please focus on harmonizing the guidance above with the guidance below, subject to our current time constraints. If there are any recommendations that we don't yet implement in our infra-live/infra-modules templates, please create separate stories for them and call them out in Slack so that we can figure out where to prioritize them in future sprints.
```

Comment:
Do you have more concrete requests other than harmonizing? Large chunks of this doc was copied from Jim's post.

Reply:
I actually didn't review the existing content in this PR in depth. My main thinking was that:

1. We should cover the key items that keep coming up, most of which I believe Jim covers:
   - How to create the backend (e.g. S3 bucket) for storing Terraform state
   - How to structure your folder hierarchy
   - How to handle tagging and labels
   - Whether to use branches per environment
   - How to handle global variables
   - How to handle module default values
1. We should make sure that our recommendations here either reflect Jim's advice, or if we disagree, do so consciously and intentionally (but to be clear, it is ok to disagree!)